eng/Publishing.props

```diff
@@ -31,13 +31,15 @@
     <_InstallersToPublish Include="$(ArtifactsDir)installers\**\*.version" UploadPathSegment="Runtime" Condition="'$(PublishInstallerBaseVersion)' == 'true'" />
 
     <!-- The following installers create checksums -->
-    <_InstallersToPublish Include="$(ArtifactsDir)installers\**\*.deb" UploadPathSegment="Runtime" ChecksumPath="%(FullPath).sha512" />
+    <_InstallersToPublish Include="$(ArtifactsDir)packages\**\*.deb" UploadPathSegment="Runtime" ChecksumPath="%(FullPath).sha512" />
```

Comment:
Is it the common pattern to publish installers to the `packages` dir?

Reply:
Yes, dotnet/runtime and dotnet/windowsdesktop publish all installers to the `packages` dir.